docs/how-to-guides/run-and-evaluate-a-flow.md

```diff
@@ -198,6 +212,24 @@ There are actions to trigger local batch runs. To perform an evaluation you can
 
 ::::
 
+## How to log metrics
+
+Promptflow supports logging and tracking experiments using `log_metric` function. A metric is a key-value pair that records a single float measure. In a python node, you can log a metric with below code: 
+
+```python
+from promptflow import log_metric, tool
+
+@tool
+def example_log_metrics():
+  metric_key = "accuracy"
+  metric_value = 1.0
+  log_metric(metric_key, metric_value)
```

Comment:
currently, our `log_metric` has one deprecated parameter `variant_id`, not sure whether we need to highlight this because there might be customer using that...

Reply:
Yeah that's a deprected parameter and we don't want user to use it. I prefer we not mention it to reduce the confusion. Even if user uses it, it only modifies the metric key name, won't cause error.